Cargo.toml

```diff
@@ -15,7 +15,7 @@ include = ["bindings/rust/*", "grammar.js", "queries/*", "src/*"]
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter = "~0.20.10"
+tree-sitter = "~0.24"
```

Comment:
```suggestion
tree-sitter = "0.24"
```

Reply:
Thanks for the feedback. I updated that part. Please don't force you into reviewing if you're not well. This is like a sunday PR when i had a bit of time. Nothing crucial to steal someones lifetime :)

If you want to give it a proper look after you are well, I'd be glad to wait.